packages/components/src/Payments/PaymentsEthForm.tsx

```diff
@@ -0,0 +1,135 @@
+    };
+  }
+
+  public render(): JSX.Element {
+    const PAY_MODAL_COMPONENTS: TransactionButtonModalContentComponentsProps = {
+      [progressModalStates.IN_PROGRESS]: <PaymentInProgressText />,
+      [progressModalStates.SUCCESS]: <PaymentSuccessText />,
+      [progressModalStates.ERROR]: <PaymentErrorText />,
+    };
+
+    return (
+      <form>
+        <PaymentEthUserInfoForm>
+          <label>Email address (optional)</label>
+          <InputValidationUI inputState={this.state.emailState} width={"500px"}>
+            <input id="email" name="email" type="email" maxLength={254} onBlur={() => this.handleOnBlur(event)} />
+          </InputValidationUI>
+        </PaymentEthUserInfoForm>
+        <PaymentNotice>
+          <PaymentEthNoticeText />
+        </PaymentNotice>
+        <TransactionButton
+          transactions={[
+            {
+              transaction: this.sendPayment,
+              handleTransactionHash: this.handleTransactionHash,
+              onTransactionError: this.onTransactionError,
+              postTransaction: this.postTransaction,
+            },
+          ]}
+          modalContentComponents={PAY_MODAL_COMPONENTS}
+        >
+          Tip
+        </TransactionButton>
+        <PaymentTerms>
+          <PaymentEthTermsText />
+        </PaymentTerms>
+      </form>
+    );
+  }
+
+  private handleOnBlur = (event: any) => {
+    const state = event.target.id;
+    const value = event.target.value;
+
+    switch (state) {
+      case "email":
+        const validEmail = isValidEmail(event.target.value);
+        validEmail || value === ""
+          ? this.setState({ email: value, emailState: INPUT_STATE.VALID })
+          : this.setState({ emailState: INPUT_STATE.INVALID });
+        break;
+      default:
+        break;
+    }
+  };
+
+  private sendPayment = async (): Promise<TwoStepEthTransaction<any> | void> => {
+    this.context.fireAnalyticsEvent("tips", "start submit ETH support", this.props.postId, this.props.usdToSpend);
+    if (this.context.civil && (window as any).ethereum) {
+      return this.context.civil.simplePayment(this.props.paymentAddress, this.props.etherToSpend.toString());
+    }
+  };
+
+  private handleTransactionHash = async (txHash: TxHash) => {
+    this.context.fireAnalyticsEvent("tips", "ETH support submitted", this.props.postId, this.props.usdToSpend);
+    await this.props.savePayment({
+      variables: {
+        postID: this.props.postId,
+        input: {
+          transactionID: txHash,
+          paymentAddress: this.props.paymentAddress,
+          fromAddress: this.props.userAddress,
+          amount: this.props.etherToSpend,
+          usdAmount: this.props.usdToSpend.toString(),
+          emailAddress: this.state.email,
+        },
+      },
+    });
+  };
+
+  private onTransactionError = (err: string) => {
+    this.context.fireAnalyticsEvent("tips", "ETH support rejected", this.props.postId, this.props.usdToSpend);
```

Comment:
Super minor, but this analytics text makes it seem like maybe the user rejected the metamask modal, which also would result in an error (unless that's being handled in `TransactionButton` modals elsewhere? if it's not it would be nice to abstract that some day) but other things could cause an error too, so I'd say "error" instead of "rejected".

Reply:
I brought this over from the Boost tracking Nick set up to keep it consistent. It would be good to check that out though and see how clear it is.